src/main/java/com/slimebot/games/PlayerGameState.java

```diff
@@ -0,0 +1,41 @@
+package com.slimebot.games;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PlayerGameState {
+    public static Map<Long, PlayerGameState> gameStates = new HashMap<>();
+    public Game game;
+
+    public PlayerGameState(Game game) {
+        this.game = game;
+    }
+
+    public static PlayerGameState getGameState(long player) {
```

Comment:
Use UserSnowflake as parameter type (also in all other methods in this class). You don't have to use Member#getIdLong everytime you use this method then because Member implements UserSnowflake. To access the id in the method you can use UserSnowflake#getIdLong